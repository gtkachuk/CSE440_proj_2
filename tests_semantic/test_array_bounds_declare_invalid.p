program testArrayBoundsDeclareInvalid;

class testArrayBoundsDeclareInvalid

BEGIN
   
VAR
   bad : ARRAY[7..2] OF integer; 
   good	 : ARRAY[44..44] OF integer;
END
.
