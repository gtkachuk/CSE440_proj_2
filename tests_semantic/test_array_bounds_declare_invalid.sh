#!/bin/sh

. ../tests_lib.inc

test_diff_sem test_array_bounds_declare_invalid.p
