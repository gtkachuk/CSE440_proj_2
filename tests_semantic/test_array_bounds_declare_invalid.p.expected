ERROR:8:Invalid array range (7,2)
Errors detected. Exiting.
